common/djangoapps/third_party_auth/settings.py

```diff
@@ -49,6 +49,7 @@ def apply_settings(django_settings):
         'third_party_auth.pipeline.associate_by_email_if_login_api',
         'social_core.pipeline.user.get_username',
         'third_party_auth.pipeline.set_pipeline_timeout',
+        'openedx.stanford.djangoapps.auth_lagunita.social.try_send_existing_lead_to_login',
```

Comment:
Run our middleware directly before the similar one upstream;
we want our redirects to take priority.